README.adoc

```diff
@@ -1,3 +1,41 @@
+= C++ example project scanned on SonarCloud using GitLab CI
+// URIs:
+:uri-qg-status: https://sonarcloud.io/dashboard?id=sc-sq-c-family-examples_linux-cmake-gitlab-ci-sc
+:img-qg-status: https://sonarcloud.io/api/project_badges/measure?project=sc-sq-c-family-examples_linux-cmake-gitlab-ci-sc&metric=alert_status
+:uri-build-status: https://gitlab.com/tomasz.kaminski/linux-cmake-gitlab-ci-sc/-/commits/main
+:img-build-status: https://gitlab.com/tomasz.kaminski/linux-cmake-gitlab-ci-sc/badges/main/pipeline.svg
+
+image:{img-build-status}[Build Status, link={uri-build-status}]
```

Comment:
Looks like this link and image are private. At least I do not have access to them (unauthenticated).

Reply:
Yup there are also not visible for me after logout? Should I just remove them?

Reply:
Can you make the project public instead?
Just like this github project

Reply:
Ok, this was caused by my attempt to rename the project to remove `s` from the name, the links still need to have `-scs` which will be confusing. I will recreate project.